app/models/manageiq/providers/ibm_cloud/inventory/parser/vpc.rb

```diff
@@ -182,12 +182,13 @@ def vm_and_template_taggings(resource, tags_inventory_objects)
   def flavors
     collector.flavors.each do |flavor|
       memory = flavor&.dig(:memory, :value)
-      memory_mb = Integer(memory) * 1024 if memory
+      disk = flavor[:disks].first&.dig(:size, :value) || 0
       persister.flavors.build(
         :ems_ref         => flavor[:name],
         :name            => flavor[:name],
         :cpu_total_cores => flavor&.dig(:vcpu_count, :value),
-        :memory          => memory_mb,
+        :memory          => memory&.gigabytes,
+        :root_disk_size  => disk&.gigabytes,
```

Comment:
needed to add this attribute in order to make the form work due to https://github.com/ManageIQ/manageiq-ui-classic/blob/5e3a5d7858224498f8fa09a01aaf268a4c00693c/app/controllers/mixins/actions/vm_actions/resize.rb#L119